src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -75,6 +75,20 @@ internal partial class Binder
                         sourceTuple.HasErrors);
                 }
 
+                // identity stackalloc conversions result in a converted node
+                // to indicate that stackalloc conversions are no longer applicable.
+                // nothing else changes
+                if (source.Kind == BoundKind.StackAllocArrayCreation)
+                {
+                    var sourceStackAlloc = (BoundStackAllocArrayCreation)source;
+                    source = new BoundConvertedStackAllocExpression(
```

Comment:
Do we have semantic model tests that check the conversion?

Reply:
I will remove this change, it seems unnecessary.

---
In reply to: [139548193](https://github.com/dotnet/roslyn/pull/22162#discussion_r139548193) [](ancestors = 139548193)